Detectors/FIT/FT0/simulation/src/Detector.cxx

```diff
@@ -413,6 +344,58 @@ TGeoVolume* Detector::SetCablesSize(int mod)
   return vol;
 }
 
+void Detector::addAlignableVolumes() const
+{
+  //
+  // Creates entries for alignable volumes associating the symbolic volume
+  // name with the corresponding volume path.
+  //
+  //  First version (mainly ported from AliRoot)
+  //
+
+  LOG(INFO) << "Add FT0 alignable volumes";
+
+  if (!gGeoManager) {
+    LOG(FATAL) << "TGeoManager doesn't exist !";
+    return;
+  }
+
+  TString volPath = Form("/cave_1/barrel_1");
+  TString symName = Form("/cave_1/barrel_1");
```

Comment:
actually, the symbolic names are supposed to be more meaningful than the path. You can, surely, use the symname the same as the path, but then what is to point of repeating the same code of the string building?